src/components/VmConsole/index.js

```diff
@@ -1,27 +1,32 @@
-import React, { useEffect, useState } from 'react'
+import React, {
+  useEffect,
+  useState,
+} from 'react'
 import { connect } from 'react-redux'
 import PropTypes from 'prop-types'
 import { push } from 'connected-react-router'
-import $ from 'jquery'
```

Comment:
One less use of jquery!  Replace the `jquery.ajax` calls with standard `fetch` and we can drop it completely.

Reply:
This is a good follow-up PR ! `fetch` does not reject on error status so we could also clean up error handling in the saga layer -> https://github.com/oVirt/ovirt-web-ui/blob/bfd825b22b9b4acc14c3309f2832c1f8bd9e6e28/src/sagas/utils.js#L76